certified-connectors/SignHost/apiDefinition.swagger.json

```diff
@@ -0,0 +1,945 @@
+                "type": "string"
+              },
+              "DaysToRemind": {
+                "description": "Amount of days before reminding the signers. -1 to disable reminders.\nIgnored if `SendSignRequest` is set to false.\nBy default your organisation's setting will be used.\n",
+                "default": 7,
+                "type": "integer"
+              },
+              "Expires": {
+                "description": "When set the signer is no longer allowed to sign the transaction after this date.",
+                "type": "string",
+                "format": "date-time"
+              },
+              "Reference": {
+                "description": "The reference of the signer.",
+                "type": "string"
+              },
+              "RejectReason": {
+                "readOnly": true,
+                "description": "The rejection reason that was given by the signer when the transaction was rejected.",
+                "type": "string"
+              },
+              "ReturnUrl": {
+                "description": "The url to redirect the user to after signing, rejecting or cancelling.",
+                "default": "https://signhost.com",
+                "type": "string"
+              },
+              "Context": {
+                "type": "object",
+                "description": "Any valid json object which we will return back to you when doing a GET on the transaction or when we send a postback."
+              },
+              "Activities": {
+                "type": "array",
+                "readOnly": true,
+                "description": "List of activities attached to this signer.\nActivities are added by signhost when a signer event occured.\n",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "Id": {
+                      "type": "string"
+                    },
+                    "Code": {
+                      "type": "integer",
+                      "enum": [
+                        101,
+                        102,
+                        103,
+                        104,
+                        105,
+                        201,
+                        202,
+                        203,
+                        301,
+                        302,
+                        303,
+                        401,
+                        402,
+                        403
+                      ],
+                      "description": "* 101 -  Invitation sent\n* 102 -  Received\n* 103 -  Opened\n* 104 -  Reminder sent\n* 105 -  Document opened, Info property contains the file id of the opened document.\n* 201 -  Cancelled\n* 202 -  Rejected\n* 203 -  Signed\n* 301 -  Signed document sent\n* 302 -  Signed document opened\n* 303 -  Signed document downloaded\n* 401 -  Receipt sent\n* 402 -  Receipt opened\n* 403 -  Receipt downloaded\n"
```

Comment:
I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).